Sources/Views/MessagesCollectionView.swift

```diff
@@ -122,6 +122,13 @@ open class MessagesCollectionView: UICollectionView {
                  forSupplementaryViewOfKind: kind,
                  withReuseIdentifier: String(describing: T.self))
     }
+    
+    /// Registers a nib with reusable view for a specific SectionKind
+    public func registerNib<T: UICollectionReusableView>(_ nib: UINib? = UINib(nibName: String(describing: T.self), bundle: nil), with headerFooterClass: T.Type, forSupplementaryViewOfKind kind: String) {
+        register(nib,
```

Comment:
@maxxx777 Sorry, I thought I commented on this PR long ago. Can you do a couple things for me so that I can merge this?

1. Can we change the method name from `registerNib` -> `register`?
2. Can we add a CHANGLEOG entry under the "Fixed" section for allowing people to use nibs with `MessageReusableView`? Please take care to follow the existing CHANGELOG entry format

Reply:
@SD10 no problem, I did the updates. Please review it.